crates/ere-hosts/src/main.rs

```diff
@@ -0,0 +1,76 @@
+//! Binary for benchmarking different Ere compatible zkVMs
+
+use anyhow::Result;
+use rayon::prelude::*;
+use std::sync::Arc;
+use std::{collections::HashMap, path::PathBuf};
+use witness_generator::generate_stateless_witness;
+use zkevm_metrics::WorkloadMetrics;
+use zkvm_interface::{Compiler, Input, zkVM};
+
+/// Main entry point for the host benchmarker
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    dotenv::dotenv().ok();
+
+    // SP1 zkVM
+    benchmark::<ere_sp1::RV32_IM_SUCCINCT_ZKVM_ELF, ere_sp1::EreSP1>(
+        "sp1",
+        concat!(env!("CARGO_WORKSPACE_DIR"), "ere-guests/sp1"),
+    )?;
```

Comment:
- ere_sp1::RV32_IM_SUCCINCT_ZKVM_ELF : This is the thing that compiles the rust code
- ere_sp1::EreSP1 : This is the thing that executes, proves and verifies
- "sp1" : This is the name of the zkvm
-  concat!(env!("CARGO_WORKSPACE_DIR"), "ere-guests/sp1") : This is where the guest program is located that we want to compile

Reply:
We can probably make this binary nicer by allowing people to do `cargo run --release --zkvm sp1 risc0` but not high priority and not blocking for this